technical-reports/color/color-type.md

```diff
@@ -1,22 +1,503 @@
+      "hex": "#ff00ff"
+    }
+  }
+}
+```
+
+### A98 RGB
+
+A98 RGB is a color space that is designed to be used in displays with a wide color gamut. It is based on the Adobe RGB color space.
+
+#### Components
+
+`[Red, Green, Blue]`
+
+- Red: A number between `0` and `1` representing the red component of the color.
+- Green: A number between `0` and `1` representing the green component of the color.
+- Blue: A number between `0` and `1` representing the blue component of the color.
+
+#### Example
+
+```json
+{
+  "Hot pink": {
+    "$type": "color",
+    "$value": {
+      "colorSpace": "a98-rgb",
+      "components": [1, 0, 1],
+      "alpha": 1,
+      "hex": "#ff00ff"
+    }
+  }
+}
+```
+
+### ProPhoto RGB
+
+ProPhoto RGB is a color space that is designed to be used in displays with a wide color gamut. It is based on the ProPhoto RGB color space.
+
+#### Components
+
+`[Red, Green, Blue]`
+
+- Red: A number between `0` and `1` representing the red component of the color.
+- Green: A number between `0` and `1` representing the green component of the color.
+- Blue: A number between `0` and `1` representing the blue component of the color.
+
+#### Example
+
+```json
+{
+  "Hot pink": {
+    "$type": "color",
+    "$value": {
+      "colorSpace": "prophoto-rgb",
+      "components": [1, 0, 1],
+      "alpha": 1,
+      "hex": "#ff00ff"
+    }
+  }
+}
+```
+
+### Rec 2020
+
+Rec 2020 is a color space that is designed to be used in displays with a wide color gamut. It is based on the Rec 2020 color space.
+
+#### Components
+
+`[Red, Green, Blue]`
+
+- Red: A number between `0` and `1` representing the red component of the color.
+- Green: A number between `0` and `1` representing the green component of the color.
+- Blue: A number between `0` and `1` representing the blue component of the color.
+
+#### Example
+
+```json
+{
+  "Hot pink": {
+    "$type": "color",
+    "$value": {
+      "colorSpace": "rec2020",
+      "components": [1, 0, 1],
+      "alpha": 1,
+      "hex": "#ff00ff"
+    }
+  }
+}
+```
+
+### XYZ-D65
+
+XYZ-D65 is a color space that is designed to be able to represent all colors that can be perceived by the human eye. It is a fundamental color space — all other spaces can be converted to and from XYZ. It is based on the CIE 1931 color space, using the D65 illuminant. XYZ is not commonly used in design tools, but is useful for color conversions.
+
+#### Components
+
+`[X, Y, Z]`
+
+- X: A number between `0` and `1` representing the X component of the color.
+- Y: A number between `0` and `1` representing the Y component of the color.
+- Z: A number between `0` and `1` representing the Z component of the color.
+
+#### Example
+
+```json
+{
+  "Hot pink": {
+    "$type": "color",
+    "$value": {
+      "colorSpace": "xyz",
+      "components": [0.5929, 0.2848, 0.9699],
+      "alpha": 1,
+      "hex": "#ff00ff"
+    }
+  }
+}
+```
+
+### XYZ-D50
+
+XYZ-D50 is similar to XYZ-D65, but uses the D50 illuminant.
```

Comment:
💯 these descriptions are very good and clear! Wasn’t expecting such an approachable document out of this but it’s a nice bonus 🙂